network-api/networkapi/wagtailpages/templatetags/bg_selector_tags.py

```diff
@@ -15,8 +15,16 @@ def yes_no(value):
 
 
 @register.filter
-def extended_yes_no(value):
-    """Converts quad-state to human readable string"""
+def extended_choice(value):
+    """Converts key from ExtendedChoiceField/ExtendedYesNoField to human readable string"""
+    if value == "Good":
+        return gettext("Good")
+    if value == "AVG":
+        return gettext("Average")
+    if value == "NI":
+        return gettext("Needs Improvement")
+    if value == "Bad":
+        return gettext("Bad")
```

Comment:
Updated this existing tag instead of creating a new one, because almost all of the GeneralProductPage's fields use the template "product_criterion_primary_info.html", which uses this tag, to render its selection.